layers/meta-tegrademo/recipes-bsp/tegrademo-devicetree/tegrademo-devicetree_1.0.bb

```diff
@@ -0,0 +1,29 @@
+DESCRIPTION = "Meta-tegrademo example device trees for out-of-tree builds."
+HOMEPAGE = "https://github.com/OE4T"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit devicetree
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+SRC_URI = "\
+    file://tegra194-p2888-0001-p2822-0000-oe4t.dts \
+    file://tegra194-p3668-all-p3509-0000-oe4t.dts \
+"
+
+DEPENDS += "linux-tegra"
```

Comment:
This isn't necessary - `devicetree.bbclass` adds an appropriate dependency for you.